src/index.ts

```diff
@@ -11,7 +11,6 @@ const escaped: Record<string, string> = {
 	'\n': '\\n',
 	'\r': '\\r',
 	'\t': '\\t',
-	'\0': '\\u0000',
```

Comment:
Sorry for being unclear earlier; I meant “why not produce the escape sequence for `\0` (i.e. `'\\0'`) instead of the long-form `\u` one”. I think escaping U+0000 makes a lot of sense and is preferable to not escaping it.